assets/css/stripeStyle.css

```diff
@@ -379,3 +559,7 @@ table tr {
 	position: absolute;
 	transform: rotate(-18deg) translateY(8px);
 }
+
+.as {
+	display: none;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)